game/scripts/npc/items/reflex_cores/react/item_reactive_3a.txt

```diff
@@ -35,7 +35,7 @@
 		"AbilityTextureName"    "custom/item_reactive_3a"
 		// Stats
 		//-------------------------------------------------------------------------------------------------------------
-		"AbilityCooldown"       "10.0"
+		"AbilityCooldown"       "0.5"
```

Comment:
Shouldn't the charges negate cooldown? If it doesn't I think the item isn't quite working like it should.